bot/seasons/valentines/be_my_valentine.py

```diff
@@ -0,0 +1,250 @@
+
+        guild = self.bot.get_guild(id=Client.guild)
+        emoji_1, emoji_2 = self.random_emoji()
+        Role = discord.utils.get(guild.roles, id=Lovefest.role_id)
+        print(Role)
+
+        if user is None:
+            author = ctx.author
+            user = self.random_user(author, Role.members)
+            if user is None:
+                return await ctx.send("There are no users avilable to whome your valentine can be sent.")
+            else:
+                pass
+            # just making sure that the random does not pick up the same user(ctx.author)
+
+        if valentine_type is None:
+            valentine, title = self.random_valentine()
+
+        elif valentine_type.lower() in ['p', 'poem']:
+            valentine = self.valentine_poem()
+            title = f'A poem dedicated to'
+
+        elif valentine_type.lower() in ['c', 'compliment']:
+            valentine = self.valentine_compliment()
+            title = f'A compliment for'
+
+        else:
+            # in this case, the user decides to type his own valentine.
+            valentine = valentine_type
+            title = f'A message for'
+
+        embed = discord.Embed(
+            title=f'{emoji_1}{title} {user.display_name}{emoji_2}',
+            description=f'{valentine} \n **{emoji_2}From anonymous{emoji_1}**',
+            color=Colours.pink
+        )
+        await ctx.author.send(f"Your message has been sent to {user}")
+        await user.send(embed=embed)
+
+    @staticmethod
+    def random_user(author, members):
+        members.remove(author)
+        if members.__len__() == 0:
+            user = None
+        else:
+            user = random.choice(members)
+        return user
+
+    @staticmethod
+    def random_emoji():
+        EMOJI_1 = random.choice(HEART_EMOJIS)
+        EMOJI_2 = random.choice(HEART_EMOJIS)
+        return EMOJI_1, EMOJI_2
+
+    def random_valentine(self):
+        """
+        Grabs a random poem or a compliment (any message).
+        """
+        valentine_poem = random.choice(self.valentines['valentine_poems'])
+        valentine_compliment = random.choice(self.valentines['valentine_compliments'])
+        random_valentine = random.choice([valentine_compliment, valentine_poem])
+        if random_valentine == valentine_poem:
+            title = 'A poem dedicated to'
+        else:
+            title = 'A compliment for '
+        return random_valentine['message'], title
+
+    def valentine_poem(self):
+        """
+        Grabs a random poem.
+        """
+        valentine_poem = random.choice(self.valentines['valentine_poems'])
+        return valentine_poem['message']
+
+    def valentine_compliment(self):
+        """
+        Grabs a random compliment.
+        """
+        valentine_compliment = random.choice(self.valentines['valentine_compliments'])
+        return valentine_compliment['message']
```

Comment:
I don't understand why all of these are dictionaries instead of strings if you're only ever using `message` anyways, or am I missing something?

Reply:
im using a json file.

Reply:
Yes, but what does that have to do with this?

Reply:
?

Reply:
In the JSON file you have objects but all of them are just in the format
```json
{
    "message": "something"
}
```
Why not have the list containing these objects contain the message directly instead?

Reply:
oh okay,got it.